src/components/Slider/Slider.vue

```diff
@@ -146,8 +144,12 @@ export default {
   },
   methods: {
     configureTooltip() {
-      this.sliderWidth = this.$refs.input.clientWidth;
-      this.labelWidth = this.$refs.tooltip.$refs.label.clientWidth;
+      if (this.$refs.input) {
+        this.sliderWidth = this.$refs.input.clientWidth;
+      }
+      if (this.$refs.tooltip && this.$refs.tooltip.$refs.label) {
+        this.labelWidth = this.$refs.tooltip.$refs.label.clientWidth || 32;
+      }
```

Comment:
This way I imagine it would be more readable and scalable:
```suggestion
      const tooltipLabel = this.$refs.tooltip?.$refs.label
      if (tooltipLabel) {
        this.labelWidth = tooltipLabel.clientWidth || 32;
      }
```